setup.py

```diff
@@ -49,7 +49,6 @@
     'pycodestyle~=2.6.0',
     'pyflakes~=2.2.0',
     'flake8~=3.8.1',
-    'uvloop>=0.15.1;platform_system!="Windows"',
```

Comment:
Can we actually keep it and install it on Python > 3.6? Can we add one more condition here? Alternatively we can make the GH CI Action to install it and then run tests with it.